src/components/schemas/fields.yaml

```diff
@@ -198,7 +198,36 @@ locations:
   minItems: 0
   maxItems: 10
   items:
-    $ref: './fields.yaml#/location'
```

Comment:
This component was used at a lot of other places so not changing parent. Changed only for expenses POST.